application/src/main/java/org/opentripplanner/graph_builder/module/osm/WalkableAreaBuilder.java

```diff
@@ -640,27 +633,21 @@ private void createNamedAreas(AreaEdgeList edgeList, Ring ring, Collection<Area>
       I18NString name = namer.getNameForWay(areaEntity, id);
       namedArea.setName(name);
 
+      WayProperties wayData;
       if (!wayPropertiesCache.containsKey(areaEntity)) {
-        WayProperties wayData = areaEntity
-          .getOsmProvider()
-          .getWayPropertySet()
-          .getDataForWay(areaEntity);
+        wayData = areaEntity.getOsmProvider().getWayPropertySet().getDataForWay(areaEntity);
```

Comment:
This way vs area thing here is slightly confusing, but I think it's a common problem that we use these way properties for areas as well so if we were to rename things, we would need to rename a lot of things.